lib/rules/comma-style.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-
+        fixable: true,
```

Comment:
I think we usually want "code" or "whitespace" here. Not sure which this is though!